libraries/lib-module-manager/PluginManager.cpp

```diff
@@ -1143,6 +1143,19 @@ ComponentInterface *PluginManager::Load(const PluginID & ID)
    return nullptr;
 }
 
+void PluginManager::ClearEffectPlugins()
+{
+   for (auto it = mRegisteredPlugins.cbegin(); it != mRegisteredPlugins.cend(); ++it)
+   {
+      auto& desc = it->second;
+      if (desc.GetPluginType() == PluginTypeEffect)
+      {
+         mLoadedInterfaces.erase(desc.GetID());
```

Comment:
Probably I was wrong when I said that effect should be erased from the `mLoadedInterfaces`. It may cause problems with effects being used in the project since they are unique ptrs. Removing description of the loaded plugin isn't good thing as well because there are code pathes that request plugin information from the `PluginManager`. Placing `PluginDescription` of the loaded plugins into special container in `PluginManager::ClearEffectPlugins` so that later calls to `PluginManager::GetPlugin`... could work